about.html

```diff
@@ -35,14 +36,15 @@ <h2 class="about-main-message-subhead">I-HEALTH OFFICIAL LAUNCH 2022</h2>
 
         <section class="about-logo-section">
             <h1 class="about-logo-section-head">I-HEALTH OFFICIAL LAUNCH 2022 LOGO</h1>
```

Comment:
- [ ] [Optional] For the `p` tag with class `about-main-message-content`, kindly make follow the design guidelines for that section to style the section on your project. The tet content is horizontally aligned and not vertically aligned. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181917056-5bea4738-81b1-4ccc-acf0-377d24c2d943.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181917088-6c9a7b67-bdec-454d-ad9a-8face1e0b536.png)